stdlib/2and3/distutils/command/install.pyi

```diff
@@ -0,0 +1,14 @@
+from distutils.cmd import Command
+from typing import Union, Optional
+
+
+class install(Command):
+    user: Union[int, bool]
+    prefix: Optional[str]
+    home: Optional[str]
+    root: Optional[str]
+    install_lib: Optional[str]
```

Comment:
The four fields above should use `Text` instead of `str` for Python 2 compatibility.